xml/System.Windows.Shell/WindowChrome.xml

```diff
@@ -480,6 +480,23 @@
         <remarks>To be added.</remarks>
       </Docs>
     </Member>
+    <Member MemberName="IsHitTestVisibleInChrome">
```

Comment:
This file has a couple new members, which seems odd.

Reply:
I've sent email to Joel/Mimi about these attached properties. They were previously documented on MSDN only.